native/membrane_video_compositor/src/compositor.rs

```diff
@@ -97,8 +98,16 @@ impl PluginArgumentEncoder for MockTransformation {
 }
 
 impl Transformation for MockTransformation {
-    fn do_stuff(&self, arg: &Self::Arg) {
-        println!("This is a mock transformation called with the string \"{arg}\" :^)")
+    type Error = ();
+
+    fn apply(
+        &self,
+        arg: &Self::Arg,
+        _source: &Texture,
+        _target: &Texture,
```

Comment:
Shouldn't this one be mutable? I guess we want to "draw onto" this texture.
```suggestion
        _target: &mut Texture,
```

Reply:
Ok, it doesn't have to be

Reply:
Draw methods do not require mutable reference